tools/mixmodel4repeated_measures/mixmodel.xml

```diff
@@ -0,0 +1,215 @@
+<tool id="mixmodel" name="mixmodel" version="3.0.0+galaxy0">
```

Comment:
@jfrancoismartin, are you sure that you want to start this tool with a version 3.0.0?
(`+galaxy0` is just a nomenclature to deal with changes in the wrapper itself)